### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by brinxmat**

### CODE REVIEW

File: import-cli/src/main/java/no/unit/nva/brage/importcli/ImportCli.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package no.unit.nva.brage.importcli;public class ImportCli {
```
Comment (janbj) [inline import-cli/src/main/java/no/unit/nva/brage/importcli/ImportCli.java]: Linebreak?

File: import-cli/src/test/java/no/unit/nva/brage/importcli/TestCli.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+public class TestCli {
+
+    @Test
+    void importCliPrintsOutput() {
+        var importCli = new ImportCli();
+        CommandLine commandLine = new CommandLine(importCli);
+        var stringWriter = new StringWriter();
+        commandLine.setOut(new PrintWriter(stringWriter));
+        assertThat()
```
Comment (janbj) [inline import-cli/src/test/java/no/unit/nva/brage/importcli/TestCli.java]: assert what?

File: import-cli/src/main/java/no/unit/nva/brage/importcli/ImportCli.java (hunk 1 above)
Comment (brinxmat) [reply import-cli/src/main/java/no/unit/nva/brage/importcli/ImportCli.java]: I fixed this in the next PR, the ImportCli and TestCli classes were included mistakenly

**Review by axthosarouris — CHANGES_REQUESTED** [review]
This PR does not build.I am not sure we want to merge to main before we merge everything to develop. This can wait until all things have been merged to develop?

**Pull request merged**